absl/logging/__init__.py

```diff
@@ -863,7 +863,7 @@ def flush(self):
     self.acquire()
     try:
       self.stream.flush()
-    except (EnvironmentError, ValueError):
+    except (EnvironmentError, ValueError, AttributeError):
```

Comment:
Instead of catching the AttributeError, let's match stdlib's behavior by checking self.stream first: https://github.com/python/cpython/blob/f34176b77f222726d901595968a4b44456186da4/Lib/logging/__init__.py#L1143

Reply:
Yes, that's better than catching any AttributeError. I updated the pull request.